master/internal/api_workspace.go

```diff
@@ -1558,8 +1558,8 @@ func (a *apiServer) GetKubernetesResourceQuotas(
 	if err != nil {
 		return nil, err
 	}
-	err = workspace.AuthZProvider.Get().CanGetWorkspaceID(
-		ctx, *curUser, req.Id,
+	err = workspace.AuthZProvider.Get().CanViewResourceQuotas(
+		ctx, *curUser,
```

Comment:
Instead of removing the `CanGetWorkspaceID` check, should we maybe do  
`a.getWorkspaceAndCheckCanDoActions(ctx, req.Id, false, workspace.AuthZProvider.Get().CanGetWorkspace, workspace.AuthZProvider.Get().CanViewResourceQuotas)` so that we perform both of these checks? Or do we not need the `CanGetWorkspaceID` check here?

Reply:
I don't think we can check 2 permissions at a time, so I added both checks in a different way